packages/vitest/src/node/git.ts

```diff
@@ -0,0 +1,95 @@
+import { resolve } from 'pathe'
+import { execa } from 'execa'
+import type { ExecaReturnValue } from 'execa'
```

Comment:
We could use `simple-git` here

Reply:
I didn't want to bring other dependencies here, since what I need can be achieved easily with a simple command and `split('\n')`, so not sure it's justified

Reply:
Sure. I am just not confident about maintaining it, since it might be a bit hard to test. No strong opinion on this.

Reply:
Are you afraid git will abandon support for this commands? I was looking at jest code for this, and they haven't changed it for 5 years 👀

Reply:
No.. I mean, in different OS, or different git versions (or even global settings), it might emit different outputs. But I might be overthinking.

Reply:
I think this is fine until we don't have reports from users.